workers/unity/Packages/com.improbable.gdk.tools/RedirectedProcess.cs

```diff
@@ -208,11 +218,25 @@ public int Run()
 
                     foreach (var errorProcessor in errorProcessors)
                     {
-                        errorProcessor(errorString);
+                        try
+                        {
+                            errorProcessor?.Invoke(errorString);
+                        }
+                        catch (Exception e)
+                        {
+                            Debug.LogException(e);
+                        }
                     }
                 };
 
                 process.EnableRaisingEvents = true;
+
+                if (!process.Start())
+                {
+                    throw new Exception(
+                        $"Failed to run {info.FileName} {info.Arguments}\nIs the .NET Core SDK installed?");
```

Comment:
Error message doesn't make sense. It may not be a `dotnet` process

Reply:
This was already there

Reply:
Please fix it then 😉